Libplanet/Blockchain/BlockChain.cs

```diff
@@ -182,6 +182,21 @@ public partial class BlockChain : IBlockChainStates
                     stored: Genesis.Hash
                 );
             }
+
+            if (Tip.ProtocolVersion < BlockMetadata.SlothProtocolVersion)
+            {
+                return;
+            }
```

Comment:
Can't we just put NSRH regardless of its protocol version? There are no safety measures anyway. We might as well make the definition of NSRH as "the executed result of a block". 😶

Reply:
Since then, name `NSRH` seems to be weird a bit(For protocol version lower than sloth). I still think about that, but suspended those kind of thinking to after 5.0.0 release.